code/scripting/scripting.cpp

```diff
@@ -1212,6 +1214,10 @@ bool script_state::IsActiveAction(ConditionalActions action_id) {
 	return ActiveActions[action_id];
 }
 
+bool script_state::IsActiveAction(int hookId) {
+	// TODO
```

Comment:
Support for this will require making `ActiveActions` dynamic, e.g. via a unorder map. The hook id is always equal or greater than `CHA_LAST` so they will not conflict with the built-in hook ids.

Reply:
Done